iga/invenio.py

```diff
@@ -257,19 +257,18 @@ def invenio_community_send(record, community):
     }
     result = _invenio('put', url=record.review_url, data=data,
                       msg='get community submission link from InvenioRDM')
-    if submit_url := result.get('links', {}).get('actions', {}).get('submit', ''):
-        data = {
+    submit_url = record.review_url.replace('/review','/actions/submit-review')
```

Comment:
Separately, why not test to make sure that a submit URL has actually been obtained? It seems better to conditionalize the network call, as was done before.

Reply:
The submit_url doesn't come from a network call anymore; I'm just hard coding it because the correct url doesn't come back from any call at the moment.